### ISSUE TITLE
How to continue the (second) training by loading the last generated (.weights) file?

### ISSUE THREAD

**Issue by leekwunfung817** [AlexeyAB/darknet#1480]
after I execute the following command
`darknet.exe detector train cfg/cpos_obj.data cfg/cpos-yolov3-tiny.cfg darknet19_448.conv.23`

![untitled](https://user-images.githubusercontent.com/18736700/44640335-ba260100-a9f3-11e8-9d97-76fc3b3537ff.png)

I am going to training until the step 840, and the last step is 800.
![untitled](https://user-images.githubusercontent.com/18736700/44640376-f5283480-a9f3-11e8-91eb-fb0d29f064e6.png)

**Comment by AlexeyAB** [comment]
* What GPU do you use?
* Can you show your [net]-section of cfg file?
* Try to increase subdivisions=32 or 64 in cfg-file

**Comment by leekwunfung817** [comment]
NVIDIA GeForce 710

[net]
batch=64
subdivisions=64
width=224
height=224
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

**Comment by leekwunfung817** [comment]
Can you tell me how to train the model(.weights) that I have been train?
I need a command. Thank you @AlexeyAB

**Comment by PeterQuinn925** [comment]
Are you asking how to train from where it left off? Find the last .weights file in the backup folder and use that for the weights.  Let's assume it's cpos-yolov3-tiny_800.weights. Your command line would be:

darknet.exe detector train cfg/cpos_obj.data cfg/cpos-yolov3-tiny.cfg backup/cpos-yolov3-tiny_800.weights

**Comment by leekwunfung817** [comment]
problem solved

**Comment by gzhengli** [comment]
The results of my train are  like yours.Obj and no-obj are all around 0.5,and the loss more than 100.The possible reason for this phenomenon is what?

**Comment by leekwunfung817** [comment]
@gzhengli the network structure is not match, there are some solution
1. Train the new network to make sure the average loss will decrease effectively.
2. Train the pre-training network that is match your (.cfg) file structure.

I have been ask the @AlexeyAB already
https://github.com/AlexeyAB/darknet/issues/1494

**Comment by gzhengli** [comment]
@leekwunfung817 Thank you for your answer very much.
I am using  yolov2-tiny-voc.cfg and  the pre-training network is darknet53.conv.74 which from the official website.
Is this okay?

**Comment by AlexeyAB** [comment]
@gzhengli For training Yolov2-tiny you should use pre-trained weights `yolov2-tiny-voc.conv.13` file.

Download default weights file for yolov2-tiny-voc: https://pjreddie.com/media/files/yolov2-tiny-voc.weights

Get pre-trained weights tiny-yolo-voc.conv.13 using command: `darknet.exe partial cfg/yolov2-tiny-voc.cfg yolov2-tiny-voc.weights yolov2-tiny-voc.conv.13 13`

Start training: `darknet.exe detector train data/obj.data yolov2-tiny-voc.cfg yolov2-tiny-voc.conv.13`
